### ISSUE TITLE
Accessibility: improve button labelling

### ISSUE THREAD

**Issue by biou** [goblindegook/littlefoot#417]
Currently the accessible name of the buttons are managed with aria-label. ARIA attributes should be used in last resort, there are other ways to naming interactive elements. 
Here is [an article from Adrian Roselli explaining his method to name controls](https://adrianroselli.com/2020/01/my-priority-of-methods-for-labeling-a-control.html).
The first proposed technique is to use native HTML techniques, in our case, we can give an accessible name to the SVG image inside the button.
One of the advantages of not relying on the aria-label attributes is that translation services will be able to translate the accessible name.

The following articles describe ways to achieve this : [accessible svgs on CSS-tricks](https://css-tricks.com/accessible-svgs/) and [accessible svg methods for adding alternative content on Tutsplus](https://webdesign.tutsplus.com/tutorials/accessible-svg-methods-for-adding-alternative-content--cms-32205)

**Comment by goblindegook** [comment]
PR looks good, thanks, just one minor change suggestion.
